jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/TimeoutException.java

```diff
@@ -5,9 +5,13 @@
 package com.magenic.jmaqs.utilities.helper.exceptions;
 
 /**
- * Custom TimeoutException, as the built in one does not allow for a passed in inner exception.
+ * Custom TimeoutException, as the built in one does not allow for a passed in
+ * inner exception.
  */
 public class TimeoutException extends Exception {
+
+  private static final long serialVersionUID = 1;
```

Comment:
comment what this variable does

Reply:
SerialVeriondUID is acceptable use in an exception

Reply:
Keeping per Jason